A91-outlier-detection-metrics.md

```diff
@@ -0,0 +1,75 @@
+A91: gRPC Metrics for Outlier Detection
+---
+* Author(s): @pardhukonakanchi, @huntsman90
+* Approver: @markdroth
+* Status: In Review
+* Implemented in: Core, Go
+* Last updated: 2025-03-05
+* Discussion at: https://groups.google.com/g/grpc-io/c/iMezDbq5U-g
+
+## Abstract
+
+This document proposes some new metrics that will be added in gRPC for xDS Client Outlier Detection.
+
+## Background
+
+[A50: gRPC xDS Outlier Detection Support][A50] is a spec for gRPC to support Outlier Detection. The current implementation only offers debug and trace logging in terms of visibility, which can be insufficient for understanding and diagnosing decision making in large-scale production systems. 
+
+Using [A79: Non-per-call Metrics Architecture][A79], it is possible to add granular metrics to make visibility into outlier detection easy for service owners utilizing gRPC.
+
+### Related proposals: 
+* [A50: gRPC xDS Outlier Detection Support][A50]
+* [A66: OpenTelemetry Metrics][A66]
+* [A79: Non-per-call Metrics Architecture][A79]
+* [A75: xDS Aggregate Cluster Behavior Fixes][A75]
+* [A89: Backend Service Metric Label][A89] (pending)
+
+[A50]: A50-xds-outlier-detection.md
+[A66]: A66-otel-stats.md
+[A75]: A75-xds-aggregate-cluster-behavior-fixes.md
+[A79]: A79-non-per-call-metrics-architecture.md
+[A89]: https://github.com/grpc/proposal/pull/471
```

Comment:
A89 has now been merged, so this link can be changed to point to `A89-backend-service-metric-label.md`.